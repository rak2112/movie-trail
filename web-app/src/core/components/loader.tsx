import React from 'react';

export const Loader = () => {
  return (
    <div className="alert alert-info" role="alert">
      <i className="glyphicon glyphicon-repeat gly-spin"/>
      <span>This may take a moment, please wait.</span>
    </div>
  );
};