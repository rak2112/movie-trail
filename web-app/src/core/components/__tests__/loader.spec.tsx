
import React from 'react';
import { create } from 'react-test-renderer';
import { Loader } from '../loader';

describe(`Loader Component`, () => {

  it(`should render`, () => {
    const wrapper = create(
      <Loader />
    ).toJSON();
    
    expect(wrapper).toMatchSnapshot();
  });
});