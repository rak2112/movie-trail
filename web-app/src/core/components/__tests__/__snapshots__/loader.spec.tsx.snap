// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader Component should render 1`] = `
<div
  className="alert alert-info"
  role="alert"
>
  <i
    className="glyphicon glyphicon-repeat gly-spin"
  />
  <span>
    This may take a moment, please wait.
  </span>
</div>
`;
